### PR TITLE
test: run unused and gosimple over all packages at once

### PR DESCRIPTION
**Pull request by heyitsanthony**
fmt pass went from ~20 CPU minutes to ~1 CPU minute.

/cc @gyuho

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -224,26 +224,22 @@ function fmt_pass {
...
 	if which gosimple >/dev/null; then
 		echo "Checking gosimple..."
-		for path in $GOSIMPLE_UNUSED_PATHS; do
-			simplResult=`gosimple ${path} 2>&1 || true`
-			if [ -n "${simplResult}" ]; then
-				echo -e "gosimple checking ${path} failed:\n${simplResult}"
-				exit 255
-			fi
-		done
+		simplResult=`gosimple ${GOSIMPLE_UNUSED_PATHS} 2>&1 || true`
```
Comment (fanminshi) [inline test]: I wonder what's use of  `|| true`?

Comment (heyitsanthony) [inline test]: The script is `set -e`; the `true` is to catch the error so the script doesn't close before reporting the error.

Comment (fanminshi) [inline test]: I see. that make sense ty.

**Comment by gyuho** [comment]
lgtm. Thanks!

**Comment by fanminshi** [comment]
lgtm

**Comment by heyitsanthony** [comment]
Multiple executors on ci vm was causing proxy to time out; set back to 1. Merging.

**Pull request merged**